app/controllers/concerns/course_basic_detail_concern.rb

```diff
@@ -116,6 +120,20 @@ def build_previous_course_creation_params
     @course_creation_params = course_params
   end
 
+  def build_meta_course_creation_params
+    @meta_course_creation_params = params
+      .except(
+        :on,
+        :provider_code,
+        :recruitment_cycle_year,
+        :utf8,
+        :course,
+        :commit,
+        :format,
+      )
+      .permit(:goto_confirmation)
```

Comment:
We aren't using these params for mass-assigns, so I don't think `permit` is doing what we want here. Technically, it could allow us to assign to `goto_confirmation` in the course model, which doesn't exist, and is confusing.

Can we pluck out the attributes we care about in a more appropriate way?